crates/forge-runner/src/contracts_data.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::Result;
+use cheatnet::runtime_extensions::forge_runtime_extension::cheatcodes::declare::get_class_hash;
+use std::collections::HashMap;
+use bimap::BiMap;
+
+use scarb_api::StarknetContractArtifacts;
+use starknet_api::core::{ClassHash, EntryPointSelector};
+
+
+
+
+
+
+#[derive(Debug, Clone)]
+pub struct ContractsData {
+    pub contracts: HashMap<String, StarknetContractArtifacts>,
+    pub class_hashes: BiMap<String, ClassHash>,
```

Comment:
Wouldn't it be good to merge those two? 🤔

Reply:
Will open another PR (this one was created by `git spr`), but imo those are like two tables in a database